Code/Mantid/Framework/Algorithms/inc/MantidAlgorithms/AddNote.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MANTID_ALGORITHMS_ADDNOTE_H_
+#define MANTID_ALGORITHMS_ADDNOTE_H_
+
+#include "MantidAPI/Algorithm.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** AddNote : TODO: DESCRIPTION
```

Comment:
Hi @ElliotAOram , I was just snooping around. Really neat! I only noticed this TODO: DESCRIPTION that I guess the class maker had inserted. You could add a short sentence here. It doesn't need to say much, as this algorithm has its RST documentation.